hermes.podspec

```diff
@@ -22,14 +24,14 @@ end
 
 Pod::Spec.new do |spec|
   spec.name        = "hermes"
-  spec.version     = "0.5.1"
+  spec.version     = "0.5.2-rc.1"
```

Comment:
I changed this to be inline with the current version in this branch, but this should likely change before the actual release?